openfeature/api.py

```diff
@@ -94,3 +105,17 @@ def add_handler(event: ProviderEvent, handler: EventHandler) -> None:
 
 def remove_handler(event: ProviderEvent, handler: EventHandler) -> None:
     _event_support.remove_global_handler(event, handler)
+
+
+def set_transaction_context_propagator(
+    propagator: TransactionContextPropagator,
+) -> None:
+    _transaction_context_propagator = propagator
+
+
+def set_transaction_context(context: EvaluationContext) -> None:
+    _transaction_context_propagator.set_transaction_context(context)
+
+
+def get_transaction_context() -> EvaluationContext:
+    return _transaction_context_propagator.get_transaction_context()
```

Comment:
Is this the right API or do we want something based on decorators/context managers?

Reply:
Could you elaborate on this? The main use case for accessing transaction context would be within the SDK when it merges context prior to flag evaluation.

Reply:
Yes, the comment should have been in the `set_transaction_context`. I was wondering if we should do something like this instead:

```python
with start_transaction_context(context):
    continue_transaction()  # context will be cleaned up after the block completes
```

Similar to the way it works in JS by taking a callback.